gammapy/datasets/map.py

```diff
@@ -1316,22 +1316,39 @@ def plot_residuals_spatial(
         ax = residuals.plot(ax, **kwargs)
         return ax
 
-    def plot_residuals_spectral(self, ax=None, method="diff", region=None, **kwargs):
+    def plot_residuals_spectral(
+        self,
+        ax=None,
+        method="diff",
+        region=None,
+        kwargs_fit=None,
+        kwargs_safe=None,
+        **kwargs,
+    ):
         """Plot spectral residuals.
 
         The residuals are extracted from the provided region, and the normalization
         used for its computation can be controlled using the method parameter.
 
+        Both the mask fit and mask safe are taken into account.
+
         The error bars are computed using the uncertainty on the excess with a symmetric assumption.
 
         Parameters
         ----------
         ax : `~matplotlib.axes.Axes`, optional
             Axes to plot on. Default is None.
         method : {"diff", "diff/sqrt(model)"}
-            Normalization used to compute the residuals, see `SpectrumDataset.residuals`. Default is "diff".
+            Normalization used to compute the residuals, see `SpectrumDataset.residuals`.
+            Default is "diff".
         region : `~regions.SkyRegion` (required)
```

Comment:
This is strange that this is required and that a default value exists... This is not consistant